docs/getting-started/install-release.md

```
@@ -178,6 +178,20 @@ source against that pg0 installation:
 ./scripts/package-release
 ```
 
+An approved campaign container must embed its full source revision before the
+release compiles:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)